data/transactions/logic/eval.go

```diff
@@ -3070,7 +3014,7 @@ func opEcdsaPkDecompress(cx *EvalContext) error {
 			return fmt.Errorf("invalid pubkey")
 		}
 	} else if fs.field == Secp256r1 {
-		x, y = unmarshalCompressed(elliptic.P256(), pubkey)
+		x, y = elliptic.UnmarshalCompressed(elliptic.P256(), pubkey)
```

Comment:
Should we retest the speed of decompress to reconsider opcode cost? Or was our implemention the same as this library function?

Reply:
The previous `(un)marshalCompressed` was copying the library implementation, so I felt like this should not change the speed.